clickhouse/create.sql

```diff
@@ -107,4 +107,5 @@ CREATE TABLE hits
     CLID INTEGER NOT NULL,
     PRIMARY KEY (CounterID, EventDate, UserID, EventTime, WatchID)
 )
-ENGINE = MergeTree;
+ENGINE = MergeTree
+SETTINGS index_granularity = 1024;
```

Comment:
This should be only for "tuned" results.

Reply:
done